src/python/gudhi/hera/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .wasserstein import wasserstein_distance
+from .bottleneck import bottleneck_distance
```

Comment:
I think you should add:
```python
__copyright__ = "Copyright (C) ..."
__license__ = "..."
__author__ = "Marc Glisse"
```
(in `src/python/gudhi/hera/__init__.py`) to be able to know:
```python
gudhi.hera.__copyright__                                                                                                                                            
## 'Copyright (C) ...'
```

Reply:
Which one should I add? The info for the interface? For hera itself? Both?

Reply:
I think you should add the one for the interface.
This is `gudhi.hera.__copyright__`, not the hera one.